ruby/extractor/src/main.rs

```diff
@@ -299,3 +315,151 @@ fn path_for(dir: &Path, path: &Path, ext: &str) -> PathBuf {
     }
     result
 }
+
+fn skip_space(content: &Vec<u8>, index: usize) -> usize {
+    let mut index = index;
+    while index < content.len() {
+        let c = content[index] as char;
+        // white space except \n
+        let is_space = c == ' ' || c >= '\t' && c <= '\r' && c != '\n';
```

Comment:
Clippy suggests this, but I'm not sure I prefer one over the other.
```suggestion
        let is_space = c == ' ' || ('\t'..='\r').contains(&c) && c != '\n';
```